build.sbt

```diff
@@ -287,13 +287,15 @@ lazy val coreTest = project
 lazy val walletServer = project
     .in(file("app/server"))
     .settings(commonSettings: _*)
+    .settings(name := "bitcoin-s-server")
     .dependsOn(
       picklers,
       node,
       chain, 
       wallet,
       bitcoindRpc
     )
+  .enablePlugins(GitVersioning)
```

Comment:
As far as I can tell, we don't actually use this plugin for anything. Versioning of our builds are handled by `sbt-dynver`.